recipes-core/images/wpe-image-container.inc

```diff
@@ -0,0 +1,15 @@
+ROOTFS_POSTPROCESS_COMMAND += "add_subuid_subgid_files;"
+
+add_subuid_subgid_files() {
+    if [ -f ${IMAGE_ROOTFS}${sysconfdir}/subuid ]; then
+        rm -rf ${IMAGE_ROOTFS}${sysconfdir}/subuid
+    fi
+    if [ -f ${IMAGE_ROOTFS}${sysconfdir}/subgid ]; then
+        rm -rf ${IMAGE_ROOTFS}${sysconfdir}/subgid
+    fi
+    touch ${IMAGE_ROOTFS}${sysconfdir}/subuid
+    echo "root:100000:65536" >${IMAGE_ROOTFS}${sysconfdir}/subuid
```

Comment:
Where that numbers come from? Can this be parametrized (like user, uid? ,gid?)

Reply:
Currently it is hardcode based on the LXC container expecting uid/gid for root. It can moved later to scripts from here to manage based on user/groups once LXC based container also get updated